index.js

```diff
@@ -1,74 +1,269 @@
-const WEIS_IN_ETH = 1e18;
-const WEIS_IN_GWEI = 1e9;
-const GWEIS_IN_ETH = 1e9;
+const decimal = require('decimal.js');
 
-const SATOSHIS_IN_BTC = 1e8;
+const ZERO = new decimal(0);
+const ONE = new decimal(1);
+const TEN = new decimal(10);
 
-const WEIS_IN_SATOSHI = WEIS_IN_ETH / SATOSHIS_IN_BTC;
+const SATOSHIS_TO_BTC_PRECISION = new decimal(8);
+const WEIS_TO_GWEI_PRECISION = new decimal(9);
+const GWEIS_TO_ETH_PRECISION = new decimal(9);
+const WEIS_TO_ETH_PRECISION = new decimal(18);
+
+const WEIS_IN_ETH = TEN.toPower(WEIS_TO_ETH_PRECISION);
+const WEIS_IN_GWEI = TEN.toPower(WEIS_TO_GWEI_PRECISION);
+const GWEIS_IN_ETH = TEN.toPower(GWEIS_TO_ETH_PRECISION);
+const SATOSHIS_IN_BTC = TEN.toPower(SATOSHIS_TO_BTC_PRECISION);
+
+const WEIS_IN_SATOSHI = WEIS_IN_ETH.dividedBy(SATOSHIS_IN_BTC);
+const MINIMUM_GWEI = ONE.dividedBy(WEIS_IN_GWEI);
+const MINIMUM_ETH = ONE.dividedBy(WEIS_IN_ETH);
+const MINIMUM_BTC = ONE.dividedBy(SATOSHIS_IN_BTC);
+
+/**
+ * Checks if the provided value is zero, returns true if it is, false otherwise.
+ * @param {decimal.Decimal.Value} value 
+ * @returns {boolean}
+ */
+const isZero = (value) => {
```

Comment:
There are no tests(direct tests) for this function. Let's add some tests for it.

Reply:
No need. This function is not exported, so there's no way to test it directly. It is being used and tested indirectly.